### DISCUSSION TITLE
Calling unity from python ide

### DISCUSSION THREAD

**Discussion by neurochen** [exodrifter/unity-python#5] [Q&A]
Thanks for the awesome work! Could you please give me a demo on how to run the unity games inside a python ide, such as spyder or jupyter?

What I want to achieve is as following: let my python control program (a big program used for other purpose) adjust the object properties in unity. If we could just design and run a game from python ide directly, that's even cooler. Thanks.

**Comment by exodrifter** [comment]
Running Unity code from Python is outside the scope of this library. IronPython, which this library depends on, allows the execution of Python scripts in a .NET environment. It doesn't also work the other way around.

If you do find a library that does this, please add a comment with a link to the repository in case others are looking for the same thing.

**Comment by exodrifter** [comment]
It looks like using [pythonnet](https://github.com/pythonnet/pythonnet) might be your best bet.
